src/app/shared/components/template/components/task-progress-bar/task-progress-bar.component.ts

```diff
@@ -85,6 +89,9 @@ export class TmplTaskProgressBarComponent
   standalone: boolean = false;
   useDynamicData: boolean;
   private dataQuery$: Subscription;
+  radius = 16; // Radius of the circle
```

Comment:
Could be good to add a comment above these three properties to indicate that they relate to the "wheel" variant